From ae1f9a479001d519d209d19da1b9319bf4d05a28 Mon Sep 17 00:00:00 2001
From: Nicolas Fella <nicolas.fella@gmx.de>
Date: Fri, 5 Feb 2021 15:19:16 +0100
Subject: [PATCH] Fix QtKeyChainConfig when building against Qt6

6 isn't 5 and thus the resulting QtKeyChainConfig would look for Qt4.

Since Qt4 was dropped we can simplify the code.
---
 QtKeychainConfig.cmake.in | 12 ++++--------
 1 file changed, 4 insertions(+), 8 deletions(-)

diff --git a/QtKeychainConfig.cmake.in b/QtKeychainConfig.cmake.in
index 26a8ca6..0c7779d 100644
--- a/QtKeychainConfig.cmake.in
+++ b/QtKeychainConfig.cmake.in
@@ -9,14 +9,10 @@ include("${CMAKE_CURRENT_LIST_DIR}/Qt@QTKEYCHAIN_VERSION_INFIX@KeychainLibraryDe
 
 include(CMakeFindDependencyMacro)
 
-if("@QTKEYCHAIN_VERSION_INFIX@" STREQUAL "5")
-    find_dependency(Qt5Core)
-    
-    if(UNIX AND NOT APPLE)
-        find_dependency(Qt5DBus)
-    endif()
-else()
-    find_dependency(Qt4 COMPONENTS QtCore)
+find_dependency(Qt@QTKEYCHAIN_VERSION_INFIX@Core)
+
+if(UNIX AND NOT APPLE)
+    find_dependency(Qt@QTKEYCHAIN_VERSION_INFIX@DBus)
 endif()
 
 set(QTKEYCHAIN_LIBRARIES "@QTKEYCHAIN_TARGET_NAME@")
